### DISCUSSION TITLE
Self-created cohort tips for Month of Learning

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#493] [Monthly Challenge]
## Self-created Cohort 

**Accountability, community, and shared experiences can make the learning experience better, clearer, and attainable. It can diffuse frustrating situations, allow for growth together, and ultimately better, by allowing members to take turns teaching, answering questions, and navigating complex ideas together and through communication, rather than navigating the path alone.**

For Virtual Coffee created cohorts, you'll not only have the support of the community, but of the learners in your group as well. So how should you get started?

#### Be the lead
Although you can accomplish the cohort experience without a leader and by sharing responsibilities, it's often much easier to have someone take the lead to start the conversation, ensure everyone feels comfortable and welcome, and to make decisions in case anything needs changed or adusted. 

One person doesn't have to assume the role the entire time. You might break it down into modules or weeks. It is acceptable to say that every person in the group needs to take a turn leading, but this needs to be determined before everyone agrees to the experience. If you'll have multiple leaders, what happens if a person drops out of the cohort or is sick? Do you reschedule? Do you skip to the next person?

#### Determine the purpose
What do you want to learn together and why do you want to learn it? Do you want to finish a course to add to your resume? Do you want to master a new skill to help you improve your chances at a promotion or landing a job?

**You're all ready to get started with group decision making!**

### Setting up your cohort
So you've started a cohort? What should you do next? Take a minute to do some group goal setting.
	- What do each of the members want to get out of the cohort?
	- Are you following a curriculum? Is the goal to complete the entire course? Or just part of the course?
	- What is the timeframe for completing the goals?

#### Determine the pace and format of your cohort
Agreeing on the format before jumping into the cohort can help to ensure the success of the group. You might not all agree on the pace and format of the cohort. That's ok. Everyone will have different needs. Unfortunately, this might cause members to leave the cohort or start their own. Again, this is ok. The goal of the cohort is to support eachother. This requires honesty and openness. If members can't support the goals at the group and know it, then they're being honest with you by leaving and your group has a better chance at success.

**Pace**
Pace will be an incredibly important factor in the health of your group. If you're working at a fast pace, you'll need to budget the time and attention to it. Part of the time must go into supporting other members. Make sure you pad time estimates to allow for discussion and Q&A sessions.

**Format**
Will you be doing synchronous communication? Async? Both? Are there expectations for when you need to communicate with your group members? Where will you be communicating? 
	- For VC, I recommend using slack for quick communication, luma for setting up synchronous meetings, and the discussion board for answering study questions, posing questions for larger discussion or support, and zoom/google meet for live interaction.
		- Set expectations for times, guidelines, etc. for live meetings.
		
		
### Synchronous meetings
It's useful to have a format for your meetings. This ensures that everyone knows how to participate and what's expected of them. We recommend a version of the traditional VC structure:

- Give the group a couple of minutes to chat and say hello before starting the meeting.
- Five minutes after the start time, give the announcements:
	- Hi, I'm {name}, I'll be the group leader today. Our goal for this group is to {repeat the group goal}. We're talking about {what subject, and the specifics for the week} today until {give a time if you've set a limit}. To find our resources/questions/communication, you can go to our {discussion board, repo, etc.}.
		- Make sure you drop any important links here.
		- If this is a VC event, please provide the following links:
			- Code of Conduct: [https://virtualcoffee.io/code-of-conduct/](https://virtualcoffee.io/code-of-conduct/) 
			- COC Violation Form: [https://virtualcoffee.io/report-coc-violation/](https://virtualcoffee.io/report-coc-violation/)
		- You should use some statement similar to this to ensure it's clear what acceptable behavior is:
			- Just some quick reminders, be considerate of everyone in the group, We are an international community, and sometimes things get lost in translation. Let's lead with kindness, and recognize that the impact of our words matters. If things feel uncomfortable in any way reach our or use a COC violation form. Our goal is to provide a safe space for everyone here, and to help ensure that we have a Code of Conduct that we take seriously. We expect everyone to abide by the COC, whether that's during an event, in slack, or in DMs.

### Asynchronous discussions and check-ins
Asynchronousity can create further opportunities for members to share their thoughts, what they've learned, and to participate in ways that are meaningful to them. This decreases the pressure for in-person events and accomodates different schedules and time zones to keep things moving.

#### Discussions
Asynchronous discussions can also provide opportunities for keeping a record and schedule, providing searchable content, and categorizing questions and topics.

**Forums**
Forums allow participants to better organize their discussions and to stay focused on a topic. Github discussions can provide a central space for addressing topics.

#### Other Resources
- [Kent C. Dodd's Learning Clubs Ideas repo](https://github.com/kentcdodds/kcd-learning-clubs-ideas) has more resources on creating a "club" as well as a list of other curriculum and templates from base_cs, EpicReact.Dev, Rustlings, TestingJavaScript, Vimfor VSCode, and Vim Alphabet.

**Comment by dominicduffin1** [comment]
I think this is great! Can I ask what Luma is?

**Reply by BekahHW** [reply]
It's a way to schedule events, get reminders, and provide zoom links for meetings. Ray uses it for his groups. I started using it for my TFJS group. I like it. It's almost a meetup alternative for study groups.

**Reply by dominicduffin1** [reply]
Thank you!
